spec/stack.js

```diff
@@ -12,11 +12,79 @@ describe('Stack', () => {
   })
 
   context('push()', () => {
-    it('pushes an element to the top of the stack.', () => {
-      const myStack = new Stack()
+    const myStack = new Stack()
 
+    it('pushes an element to the top of the stack.', () => {
       expect(() => myStack.push('foo'))
         .to.alter(() => myStack.length(), { from: 0, to: 1 })
+      expect(myStack.top.data)
+        .to.eql('foo')
     })
   })
+
+  context('pop()', () => {
+    const myStack = new Stack()
+    myStack.push('foo')
+    myStack.push('bar')
+
+    it('removes the top element in the stack', () => {
+      expect(() => myStack.pop())
+        .to.alter(() => myStack.length(), { from: 2, to: 1 })
+    })
+    it('returns the top element in the stack', () => {
+      //at this point, bar should be removed, thus top of the stack is foo
```

Comment:
Thank you for this comment, but the tests do indicate this functionality quite clearly.